EventEmitter.js

```diff
@@ -91,14 +91,9 @@
      * @return {Function[]} Just the listener functions.
      */
     proto.flattenListeners = function flattenListeners(listeners) {
-        var flatListeners = [];
-        var i;
-
-        for (i = 0; i < listeners.length; i += 1) {
-            flatListeners.push(listeners[i].listener);
-        }
-
-        return flatListeners;
+        return listeners.map(function(l) {
+            return l.listener;
+        });
```

Comment:
This is making assumptions about the host environment. I am concerned this will break support for many people that currently use this across multiple legacy browsers. I don't think I'll be able to merge this I'm afraid :/